app/javascript/dashboard/i18n/locale/en/generalSettings.json

```diff
@@ -21,6 +21,25 @@
         "LABEL": "Site language (Beta)",
         "PLACEHOLDER": "Your account name",
         "ERROR": ""
```

Comment:
we should probably add some in-product explanation text to let the end-user know the significance of these settings cc:  @pranavrajs

Reply:
Added a documentation task for this : https://github.com/chatwoot/chatwoot/issues/778